### ISSUE TITLE
Translations from AMOS are not fully shown

### ISSUE THREAD

**Issue by FelixDiLenarda** [donhinkelman/moodle-block_sharing_cart#223]
I found that some german translations are not shown in the block.

I installed a vanilla moodle with sharing cart and boost theme.

I installed the german language pack.

Even so the lang strings are translated in AMOS, the strings are not shown in german in the block:
e.g.
![image](https://github.com/user-attachments/assets/2f179bfc-a891-42a5-a601-94d22221c77d)
and
![image](https://github.com/user-attachments/assets/e11d6311-a8a7-4989-babc-a16fb8034c34)

You can see in the picture  that the following is translated:
![image](https://github.com/user-attachments/assets/32967612-2f6b-482c-91a0-309602584881)

So some Strings are shown correctly in german and some don't.
https://lang.moodle.org/local/amos/view.php

Maybe somebody can help?
Am i missing something?
Thx :)

**Comment by FelixDiLenarda** [comment]
> [@FelixDiLenarda](https://github.com/FelixDiLenarda) is this still an issue? I just double checked and the string isn't hardcoded. - It is using a lang string that should be able to be overridden by AMOS.

@frederikmillingpytlick Yes the translations are still not working for German lang pack.

I found that many of the lang-strings are not there in the AMOS translator.
I postet this question with further details in the AMOS forum to understand how it comes:
https://lang.moodle.org/mod/forum/discuss.php?d=9515

**Comment by koenr** [comment]
I have seen the import of strings in Amos failing before, caused by not following the coding rules on https://moodledev.io/general/community/plugincontribution/checklist#strings

Causing the import to fail should also generate a notification to developers.

But I don't see any problems in this case. I wonder if something is wrong on the Amos side ...

**Comment by koenr** [comment]
Ah, here is someone with a similar problem
https://tracker.moodle.org/projects/MDLSITE/issues/MDLSITE-7945

The reason is a string using more then one line: $string['no_items'] .
It's not properly documented in the link I previously mentioned.

The plugin maintainer should see the error on https://moodle.org/plugins/mod_publication/devzone

**Comment by mblcollecto** [comment]
Hello, I have a similar issue where strings are not available on AMOS for translation. These are the same as Felix: 

<img width="484" height="294" alt="Image" src="https://github.com/user-attachments/assets/a6b6e451-4750-45fc-954a-157215b178ba" />

Strings: confirm_backup, modal_checkbox, modal_checkbox_anonymize, modal_confirm_backup

**Comment by frederikmillingpytlick** [comment]
> Hello, I have a similar issue where strings are not available on AMOS for translation. These are the same as Felix:
> 
> <img alt="Image" width="484" height="294" src="https://private-user-images.githubusercontent.com/156699705/476639485-a6b6e451-4750-45fc-954a-157215b178ba.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTkyMTIzOTcsIm5iZiI6MTc1OTIxMjA5NywicGF0aCI6Ii8xNTY2OTk3MDUvNDc2NjM5NDg1LWE2YjZlNDUxLTQ3NTAtNDVmYy05NTRhLTE1NzIxNWIxNzhiYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkzMFQwNjAxMzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMzcxYWJhOWI0NzY0N2E5Nzg1MWFiNTE2YzEzYTI3ZDhkY2YyN2M0Y2Q4NTA5NWEwYzBhMzY4ZWUzZWEzNTUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.8kiPiGVlmCP_8e5WmKDT4toQnK4qa0XGHSyDSKDCHts">
> Strings: confirm_backup, modal_checkbox, modal_checkbox_anonymize, modal_confirm_backup

This is an old version of the sharing cart.

We should be following all rules for AMOS translation to be possible. If some strings are still missing in your languages, you'd need to update it in AMOS.

I'll be closing this issue
